### PR TITLE
feat: add sentry for monitoring swap error

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
+if (process.env.REACT_APP_MAINNET_ENV === 'production') {
+  Sentry.init({
+    dsn: 'https://0b4e0f2a75a04734ad380be0a8882886@o302165.ingest.sentry.io/6156652'
```
Comment (piavgh) [inline src/index.tsx:73]: please put this to env files.

Also, the URL is not correct

File: src/index.tsx [hunk 2]
```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
+if (process.env.REACT_APP_MAINNET_ENV === 'production') {
```
Comment (viet-nv) [inline src/index.tsx:71]: change this base on domain, because auto deployment domain also use production env

Comment (viet-nv) [reply src/index.tsx:71]: For testing, we can use this for current adpr domain, but before merge, please change it to window.location.href.include("kyberswap.com")

File: src/index.tsx [hunk 3]
```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
```
Comment (viet-nv) [inline src/index.tsx:70]: is this will be same for each session? 

btw, can u move it to constant?

File: src/utils/sentry.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import * as Sentry from '@sentry/react'
+
+export function reportException(e: any) {
+  process.env.REACT_APP_MAINNET_ENV === 'production' && Sentry.captureException(e)
```
Comment (viet-nv) [inline src/utils/sentry.ts:4]: Same as index.tsx, change this base on domain check

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 1 above)
Comment (nguyenhuudungz) [reply src/index.tsx]: Updated.

File: src/index.tsx (hunk 2 above)
Comment (nguyenhuudungz) [reply src/index.tsx:71]: Why is `window.location.href.include("kyberswap.com")`, I though it was `dmm-interface-pr`.

File: src/index.tsx (hunk 3 above)
Comment (nguyenhuudungz) [reply src/index.tsx:70]: No, each session has new id.

Moved to constants.

File: src/index.tsx (hunk 2 above)
Comment (viet-nv) [reply src/index.tsx:69]: I mean before you merge this PR to master, change it to kyberswap.com
And now for testing, this is ok

File: src/index.tsx (hunk 3 above)
Comment (viet-nv) [reply src/index.tsx]: LOL. same for each session, I mean It only change when user refresh

File: src/index.tsx (hunk 2 above)
Comment (piavgh) [reply src/index.tsx:69]: @nguyenhuudungz : pls resolve this issue

**Review by viet-nv — APPROVED** [review]

**Pull request merged**